src/etc/schema.graphqls

```diff
@@ -127,10 +125,13 @@ type PaymentMethod {
     title: String
 }
 
-type PaymentTotals implements TotalsObject {
-}
-
 interface TotalsObject @typeResolver(class: "\\ScandiPWA\\QuoteGraphQl\\Model\\Resolver\\TotalsObject") {
+    items_count: Int
+    customer_is_guest: Boolean
+    store_id: Int
+    customer_tax_class_id: Int
+    items_qty: Float
+    coupon_code: String
```

Comment:
Additional improvements? Are these related to the feature?

Reply:
Is this related to previous comment?

Another note: I have just moved fields from child to interface. It will still work. Only custom field like `is_virtual` is left in the interface.